aldryn_newsblog/templates/aldryn_newsblog/includes/meta.html

```diff
@@ -0,0 +1,51 @@
+{% load meta_extra %}
+{% if meta %}
+    {% autoescape off %}
+	{% if meta.description %}{% meta 'description' meta.description %}{% endif %}
```

Comment:
Yes, I know, but I've made small changes and thought it would be necessary to make further changes in the future.

Moreover, this creates a dependency with DjangoCMS-Page-Meta for something for which it was not intended.